src/credentials.ts

```diff
@@ -52,12 +53,12 @@ export class IamAuthService extends GrpcService<IamTokenService> implements IAut
     private tokenTimestamp: DateTime|null;
     private readonly iamCredentials: IIamCredentials;
 
-    constructor(iamCredentials: IIamCredentials, useSsl: boolean = true) {
+    constructor(iamCredentials: IIamCredentials, sslCredentials?: ISslCredentials) {
         super(
             iamCredentials.iamEndpoint,
             'yandex.cloud.iam.v1.IamTokenService',
             IamTokenService,
-            useSsl
+            sslCredentials || makeSdkSslCredentials(),
```

Comment:
Is it still possible here to open grpc channel (i.e., by not passing `sslCredentials` into constructor)?

Reply:
I suppose that it's possible, but I don't understand the motivation to do it.
I don't try to change in v3 all signatures including GrpcService and AuthenticatedService constructors, so I don't have any reason to extract grpc channel initialization to extra level.

Reply:
The single reason to do it is hypothetical grpc connection to IAM (IAM is grpcs-only now).
I suggest to use null/false value for this case: `new IamAuthService(iamCredentials, false) or new IamAuthService(iamCredentials, null)` when IAM supports non-grpcs installations.
It's better then use raw grpcChannel instead of sslCredentials and in IamAuthService and Driver settings.